fatturae/settings.py

```diff
@@ -70,7 +70,14 @@
 
 DATABASES = {
     # read os.environ['DATABASE_URL'] and raises ImproperlyConfigured exception if not found
-    "default": env.db()
```

Comment:
Can you revert back to env.db()? This project uses django-environ (https://github.com/joke2k/django-environ) to make the configuration of the project dynamic via env variables. If you want to make project run on your computer create a `.env` file in the root of the project and add (for example):

`DATABASE_URL=postgres://marco@localhost/database`

this means that django will use a postgres database with the user `marco` and the database `database`.

Reply:
Sorry, this file was not meant to be committed.